tool/tsh/tsh.go

```diff
@@ -1007,6 +1075,55 @@ func showApps(servers []services.Server, verbose bool) {
 	}
 }
 
+func showDatabases(servers []services.DatabaseServer, active []tlsca.RouteToDatabase, verbose bool) {
+	if verbose {
```

Comment:
nit: these are two different implementations of showDatabases

Reply:
Yeah, I think they're pretty short as it is so I don't think it hurts readability.